Observer/Order.php

```diff
@@ -30,28 +32,23 @@ public function execute(Observer $observer)
         try {
             $order = $observer->getEvent()->getOrder();
             $storeId = $order->getStoreId();
-
+            
             if (!$this->helper->isEnabled($storeId)) {
                 return;
             }
-
-            $this->helper->callBatchApi($storeId, [$order]);
-
-            // If order is made from the FrontEnd
-            if ($order->getRemoteIp()) {
-                $identify = array(
-                    'id' => $order->getCustomerEmail(),
-                    'params' => array(
-                        'email'      => $order->getCustomerEmail(),
-                        'first_name' => $order->getBillingAddress()->getFirstname(),
-                        'last_name'  => $order->getBillingAddress()->getLastname(),
-                        'name'       => $order->getBillingAddress()->getName()
-                    )
-                );
-                $this->helper->addSessionEvent('identify', 'identify', $identify);
-            }
+            
```

Comment:
Didn't you say there must be a check making sure that the customer has an email?

Reply:
Correct, i have removed it because email check was added to the db query but forgot that the observer order is not extracted from db. Will add it again.